services/workflows-service/src/workflow/schemas/zod-schemas.ts

```diff
@@ -12,18 +12,19 @@ export const SubscriptionSchema = z.discriminatedUnion('type', [
 
 export const ConfigSchema = z
   .object({
+    isCaseOverviewEnabled: z.boolean().optional(),
+    isLegacyReject: z.boolean().optional(),
+    isLockedDocumentCategoryAndType: z.boolean().optional(),
+    isDemo: z.boolean().optional(),
+    isExample: z.boolean().optional(), // OSS only
```

Comment:
`isCaseOverviewEnabled` is the new flag, the rest are just re-ordered to be grouped by name.